build/common.sh

```diff
@@ -85,46 +85,6 @@ readonly KUBE_RSYNC_PORT="${KUBE_RSYNC_PORT:-}"
 # mapped to KUBE_RSYNC_PORT via docker networking.
 readonly KUBE_CONTAINER_RSYNC_PORT=8730
 
-# Get the set of master binaries that run in Docker (on Linux)
-# Entry format is "<name-of-binary>,<base-image>".
-# Binaries are placed in /usr/local/bin inside the image.
-#
-# $1 - server architecture
-kube::build::get_docker_wrapped_binaries() {
```

Comment:
> This is backwards incompatible, including removing release artifacts. It's likely we'll need to add back in those for a full make-release to avoid breakages.

Yeah, aware. That's part of what I meant by WIP